src/ui/linkformview.js

```diff
@@ -106,24 +144,23 @@ export default class LinkFormView extends View {
 			}
 		} );
 
+		const classList = [ 'ck', 'ck-link-form' ];
+
+		if ( this.manualDecorators.length ) {
+			classList.push( 'ck-link-form_decorators' );
```

Comment:
Maybe let's make it more generic like `ck-link-form_layout-vertical`? There could be other things in the future that will toggle the layout to vertical while not being decorators.

If not, at least change it to `ck-link-form_with-decorators` please.

Reply:
Name of class in `theme-lark` is also corrected.